chat-components/src/components/outofofficehourspane/OOOHPane.tsx

```diff
@@ -26,15 +26,15 @@ function OOOHPane(props: IOOOHPaneProps) {
                     id={elementId}
                     tabIndex={-1}
                     styles={containerStyles}
-                    role={props.controlProps?.role}
+                    role="alert"
```

Comment:
the use of props allows customization, by removing it you are removing as well that ability, just a defaultProperties value in case props.controlProps.role is null, use the default Value